pkg/server/server.go

```diff
@@ -134,14 +134,17 @@ func (s *Server) GetEventsWG(request *tetragon.GetEventsRequest, server tetragon
 	}).Debug("Received a GetEvents request")
 	allowList, err := filters.BuildFilterList(s.ctx, request.AllowList, filters.Filters)
 	if err != nil {
+		readyWG.Done()
```

Comment:
do we need to check if `readyWG` is nil? like here => https://github.com/cilium/tetragon/pull/2119/files#diff-104a11712684e520f0affa95e6a053faeb713306a10f69ee99b9729cc9dd1996R157-R159

Reply:
Probably right, let me fix that